instrumentation-api-semconv/src/main/java/io/opentelemetry/instrumentation/api/instrumenter/http/CapturedHttpHeadersUtil.java

```diff
@@ -38,8 +38,8 @@ static AttributeKey<List<String>> responseAttributeKey(String headerName) {
   }
 
   private static AttributeKey<List<String>> createKey(String type, String headerName) {
-    // headerName is always lowercase, see CapturedHttpHeaders
-    String key = "http." + type + ".header." + headerName.replace('-', '_');
+    // headerName is always lowercase, see CapturedHttpHeadersUtil#lowercase
+    String key = "http." + type + ".header." + headerName;
```

Comment:
should have put that last comment here instead ^^

Reply:
What comment exactly?

Reply:
probably
> would it be easy to handle new/old semconv here?